modules/microshift-gathering-sos-report.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+
+// * microshift_support/microshift-sos-report
+
+:_content-type: PROCEDURE
+[id="gathering-data-microshift-sos-report_{context}"]
+= Gathering data from a {product-title} sos report
+
+.Prerequisites
+
+* You must have the `sos` package installed.
+
+.Procedure
+
+. Log into the failing host as a root user.
+
+. Perform the debug report creation procedure by running the following command:
++
+[source,terminal]
+----
+$ sos report --upload --batch --clean --all-logs --profile microshift
```

Comment:
```suggestion
$ sos report --batch --clean --all-logs --profile microshift
```

Based on @vwalek 's comments, I think we want to remove the upload option that I had suggested adding previously.

Reply:
updated. Thanks!